woocommerce/woocommerce-gutenberg-products-block
# Move `Title` to components package

## What

Move `Title` component to components package and update the places it is imported:

- `FormStepHeading`
- `CheckoutExpressPayment`
- `FormStep`

## Why


As more developers integrate their extensions with WooCommerce Blocks, the case for reusing components grows. This starts the ball rolling by creating a new package. Rather than recreating components developers can access `wc.blocksComponents` and get the ones we need.


## Testing Instructions



_Please consider any edge cases this change may have, and also other areas of the product this may impact._

1. Set up [WooCommerce Stripe Payment Gateway](https://wordpress.org/plugins/woocommerce-gateway-stripe/) and ensure your site is using HTTPS so express methods from Stripe can show.
2. Open the Checkout page in the page editor. Ensure the form steps show up correctly with the correct titles.
3. Add an item to your cart.
4. Go to the Checkout block, ensure each step on the form has a correctly displaying title.
<img width="493" alt="image" src="https://github.com/woocommerce/woocommerce-blocks/assets/5656702/49c3ff6a-11f5-4a3d-b24f-599bb962868e">

6. Ensure the express payments method section has a correctly displayed title.
<img width="471" alt="image" src="https://github.com/woocommerce/woocommerce-blocks/assets/5656702/9ea746e2-20e2-4b0f-9f99-07eebb233142">

7. Check out and ensure it works.


## WooCommerce Visibility


Required:


## Checklist

Required:

Conditional:

## Changelog


> Make the `Title` component available in the `@woocommerce/blocks-components` package.
